### PR TITLE
Liform Bundle

### PR DESCRIPTION
**Pull request by GuiEloiSantos**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by GuiEloiSantos** [comment]
I thinking I'm facing the same scenario encountered at #163

**Comment by Nyholm** [comment]
Thank you. 

I do not see that this is similar to #163. Your bundle follows the best practice file structure with means that Flex can install it without a recipe.

**Comment by GuiEloiSantos** [comment]
@Nyholm you sure?

it composer has [type: library](https://github.com/Limenius/LiformBundle/blob/master/composer.json)

And even in it readme file is instructing the users to include it on app/AppKernel.php .

**Comment by Nyholm** [comment]
Im sure. Also, why does it have type: library? That should be changed since it clearly is a bundle, right?

**Comment by GuiEloiSantos** [comment]
@Nyholm it should, I'm not a maintainer of this bundle though, and I thought I could use recipes so I don't have to include the bundle manually every time I install it.

**Comment by Nyholm** [comment]
@symfony-flex-server review please

File: limenius/liform-bundle/0.12/manifest.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "Limenius\\LiformBundle\\LimeniusLiformBundle": [
```
Comment (Nyholm) [inline limenius/liform-bundle/0.12/manifest.json]: You could do this on one line

File: limenius/liform-bundle/0.12/manifest.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        "all"
+        ]
+    },
+    "aliases": [
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply limenius/liform-bundle/0.12/manifest.json]: Could you remove this please.

**Pull request closed without merging**